centreon/tests/api/features/ServiceCategory.feature

```diff
@@ -129,3 +129,134 @@ Feature:
         }
     }
     """
+
+  Scenario: Delete a service category as admin user
+    Given I am logged in
+    And the following CLAPI import data:
+        """
+        SC;ADD;service-cat1;service-cat1-alias
+        """
+
+    When I send a GET request to '/api/latest/configuration/services/categories?search={"name":{"$lk":"service-cat%"}}'
+    Then the response code should be "200"
+    And the json node "result" should have 1 elements
+
+    When I send a DELETE request to '/api/latest/configuration/services/categories/5'
+    Then the response code should be "204"
+
+    When I send a GET request to '/api/latest/configuration/services/categories?search={"name":{"$lk":"service-cat%"}}'
+    Then the response code should be "200"
+    And the json node "result" should have 0 elements
+
+    Given the following CLAPI import data:
+        """
+        SC;ADD;service-sev1;service-sev1-alias
+        SC;setseverity;service-sev1;1;logos/centreon.png
+        """
+
+    When I send a GET request to '/api/latest/configuration/services/categories?search={"name":{"$lk":"service-cat%"}}'
+    Then the response code should be "200"
+    And the json node "result" should have 0 elements
+
```

Comment:
Reminder to not forget to add here the test on service severities.

Reply:
It will be added in the PR relating to FindServiceSeverities